http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java

```diff
@@ -33,6 +35,18 @@ public abstract class AbstractVertxIT {
     @QuarkusApplication
     static final RestService service = new RestService();
 
+    Vertx vertx;
+    HttpClient httpClient;
+    @BeforeEach
+    public void setup(){
+        vertx = Vertx.vertx();
+    }
+
+    @AfterEach
+    public void teardown(){
+        httpClient.close();
+        vertx.close();
+    }
     @Test
     public void httpServerAndMetrics() {
         requests().get("/hello").then()
```

Comment:
Some tests like this one do not assign `httpClient` and:

- we can't rely on order because these tests are not ordered
- you can only run one tests, in which case you will get NPE when it doesn't create `httpClient`. Please add `null` check before you try to use `httpClient` and for my calm heart even to `vertx` though it should only happen if Vert.x instantiation failed or something awkward? anyway, it should be quick

thanks again